portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +25,27 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private String name;
+  private String comment;
+
+  @Override
+  public void init() throws ServletException {
```

Comment:
Do you need the init method right now? Its not doing much. 
HttpServlet, the base class for DataServlet,  already has an init method that throws ServletException.

Also, name and comment would by default be initialized to empty string, which should  be good enough.